guides/source/active_support_core_extensions.md

```diff
@@ -164,7 +164,13 @@ Active Support provides `duplicable?` to programmatically query an object about
 false.duplicable? # => false
 ```
 
-By definition all objects are `duplicable?` except `nil`, `false`, `true`, symbols, numbers, class, module, and method objects.
+<<<<<<< HEAD
+By definition all objects are `duplicable?` excepting method objects, numbers as well as `nil`, `false`, `true`, symbols, class, and module.
+
+As of Ruby version 2.4.0 numbers, `nil`, `false`, `true`, symbols, class and module objects are also `duplicable?`.
+=======
+By definition all objects are `duplicable?` including BigDecimals, but excepting all other numbers as well as `nil`, `false`, `true`, symbols, class, module, and method objects. As of Ruby 2.4.0 even class and module objects are `duplicable?`.
+>>>>>>> d738fd8... Update list of `duplicable?` objects in AS core_ext guide [ci skip]
```

Comment:
Thank you @kaspth , pleasure to dig in a little more.

I've learned that the above *immediate*s still can't be properly duplicated, 2.4 just quietly returns the original object. But I doubt that the core extensions guide is the place to go into this, so I think I'd remove the bit about the general logic of why singletons can not be duplicated.

`Complex` and `Rational` still won't `dup` even in 2.4.0-rc1, but return true on `duplicable?` So I'm putting this on hold for #27424 to be resolved.